readme.md

```diff
@@ -412,8 +440,12 @@ which compounds the point further. However it's worth noting that this based on
 uses a sort of nightly-build V8 version - it may not end up being a concrete characteristic in V8 6.1.
 
 If we're writing code that needs to be optimal, that is a function that will be called many times over,
-then we should avoid using polymorphism. On the other hand, if it's only called once or twice, say an instantiating/setup function,
-then a polymorphic API is acceptable.
+then we should avoid using polymorphism. On the other hand, if it's only called once or twice, say an
+instantiating/setup function, then a polymorphic API is acceptable.
+
+_Edit: We have been informed by the V8 team that the results for this specific benchmark are not
+reliably reproducible using their internal executable, `d8`.
```

Comment:
However they *are* reproducible on Node. So the results and subsequent analysis should be taken with the view that this may change in future Node versions (based on how Node is integrating with V8).